pkg/ccl/streamingccl/streamingest/stream_ingestion_job.go

```diff
@@ -510,12 +510,13 @@ func maybeRevertToCutoverTimestamp(
 	if p.ExecCfg().StreamingTestingKnobs != nil && p.ExecCfg().StreamingTestingKnobs.OverrideRevertRangeBatchSize != 0 {
 		batchSize = p.ExecCfg().StreamingTestingKnobs.OverrideRevertRangeBatchSize
 	}
+	p.ExecCfg().JobRegistry.MetricsStruct().StreamIngest.(*Metrics).ReplicatedTimeSeconds.Update(0)
```

Comment:
nit: could you add a comment explaining why this line is here, and in OnFailOrCancel?

Reply:
Yup, has been added